out

```diff
@@ -0,0 +1 @@
+This should appear in output
```

Comment:
What's this file for?

Reply:
Should not have been committed. I'll fix it.